src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -146,12 +175,24 @@ export class TaskListBuilder {
           false
         )
       )
-      return new TaskList(3, 'Submit', tasks)
+      return new TaskList(TaskListBuilder.calculateSubmitSectionNumber(draft), 'Submit', tasks)
     }
 
     return undefined
   }
 
+  private static isPartiallyAdmittedAndWhyDoYouDisagreeTaskCompleted (draft: ResponseDraft): boolean {
+    return draft.isResponsePartiallyAdmitted() && WhyDoYouDisagreeTask.isCompleted(draft)
+  }
+
+  private static calculateSubmitSectionNumber (draft: ResponseDraft): number {
```

Comment:
can we get rid of this and show the number with css instead?

Reply:
ok

Reply:
done